src/main/java/ch/njol/skript/sections/SecCatchErrors.java

```diff
@@ -0,0 +1,76 @@
+package ch.njol.skript.sections;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.ExprCaughtErrors;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Section;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.registrations.Feature;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.lang.experiment.ExperimentSet;
+import org.skriptlang.skript.lang.experiment.ExperimentalSyntax;
+import org.skriptlang.skript.log.runtime.RuntimeError;
+import org.skriptlang.skript.log.runtime.RuntimeErrorCatcher;
+
+import java.util.List;
+
+@Name("Catch Runtime Errors")
+@Description("Catch any runtime errors produced by code within the section. This is an in progress feature.")
+@Example("""
+	catch runtime errors:
+		set worldborder center of {_border} to location(0, 0, NaN value)
+	if last caught runtime errors contains "Your location can't have a NaN value as one of its components":
+		set worldborder center of {_border} to location(0, 0, 0)
+	""")
+@Since("INSERT VERSION")
+public class SecCatchErrors extends Section implements ExperimentalSyntax {
+
+	static {
+		Skript.registerSection(SecCatchErrors.class, "catch [run[ ]time] error[s]");
+	}
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult, SectionNode sectionNode, List<TriggerItem> triggerItems) {
+		if (sectionNode.isEmpty()) {
+			Skript.error("A catch errors section must contain code.");
+			return false;
+		}
+		Kleenean previousDelay = getParser().getHasDelayBefore();
+		loadCode(sectionNode);
+		if (getParser().getHasDelayBefore() != previousDelay) {
+			Skript.error("Delays can't be used within a catch errors section.");
+			return false;
+		}
```

Comment:
This won't guarantee that a delay hasn't been used. For example:
```
wait 5 seconds
catch errors:
    wait 3 seconds
```
Since `getHasDelayBefore()` and `previousDelay` will be `TRUE`. Instead, you should:
1. Cache the current delay state
2. Set the delay state to FALSE
3. load the code
4. Check if the delay state is TRUE. If it is, error and return false. Otherwise, continue on
5. Reset the delay state (to the original one cached)